src/handlers/public/send-transfer.js

```diff
@@ -16,9 +16,10 @@ module.exports = ({ plugin, prefix, routingTable, internalUri, uuidSecret, ilpEr
   }
 
   const nextHop = routingTable.getNextHop(data.account)
+  const finalAmount = new BigNumber(data.amount)
   let nextAmount = nextHop.curveLocal.amountAt(transfer.amount)
-  if (nextHop.local) {
-    const finalAmount = new BigNumber(data.amount)
+  // Always forward when finalAmount is 0.
+  if (nextHop.local && !finalAmount.equals(0)) {
```

Comment:
I think it's fine to just do `data.amount !== '0'`. Any advantage to using BigNumber here?